src/libPMacc/include/algorithms/math/floatMath/pow.tpp

```diff
@@ -43,6 +43,7 @@ struct Pow<float, float>
     HDINLINE result operator()(const float& base, const float& exponent)
     {
 #ifdef __CUDA_ARCH__ /*device version*/
+        /* @todo: CUDA has no pow(float, float) function yet but uses the C99 powf(float, float) */
```

Comment:
`CUDA 7` defines only a Runtime compile version of `pow(float,int)` and not for normal compiles, the special `pow(float,int)` case is disabled for GNU compiler

math_functions.h:9045
```C++
#if !(defined(__GNUC__) && __cplusplus >= 201103L)
extern __host__ __device__ __cudart_builtin__ float    __cdecl pow(float, int) throw();
extern __host__ __device__ __cudart_builtin__ double   __cdecl pow(double, int) throw();
#endif /* !(defined(__GNUC__) && __cplusplus >= 201103L) */
```

Now I am confused :tired_face: